Simplified.xcodeproj/project.pbxproj

```diff
@@ -511,7 +525,6 @@
 		73FCA2E825005BA4001B0C5D /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = A823D81C192BABA400B55DE2 /* main.m */; };
 		73FCA2E925005BA4001B0C5D /* NYPLCaching.swift in Sources */ = {isa = PBXBuildFile; fileRef = 733875662423E540000FEB67 /* NYPLCaching.swift */; };
 		73FCA2EA25005BA4001B0C5D /* UIView+NYPLViewAdditions.m in Sources */ = {isa = PBXBuildFile; fileRef = 1107835D19816E3D0071AB1E /* UIView+NYPLViewAdditions.m */; };
-		73FCA2EB25005BA4001B0C5D /* NYPLSettingsPrimaryTableViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = 111E75821A815CFB00718AD7 /* NYPLSettingsPrimaryTableViewController.m */; };
```

Comment:
Is there a reason why we remove the reference but not removing the file? or this is a clean up?

Reply:
that might be related to removing it from the OE target, since that uses the swift version. But yeah the reference is still there for SimplyE.